Rakefile

```diff
@@ -60,10 +60,13 @@ def tf_remote(deploy)
   desc 'configure remote state'
 
   task 'remote' do
-    remote = 'remote config -backend=s3' +
+    remote = 'init' +
+      " -backend=true" +
```

Comment:
It is a change between tf 0.8.x and 0.9.x.  Backends are now a configured resource (in code) but you can continue to pass in the attribute values.